### PR TITLE
Create .env

### PR DESCRIPTION
**Pull request by CoreValExtreme**
VAL.Core//const ALGOD_CONFIG = {
  algodToken: {
    'X-API-Key': process.env.API_KEY || ''
  },
  algodServer: process.env.ALGO_SERVER || '',
  indexerServer: process.env.INDEX_SERVER || '',
  algodPort: process.env.ALGO_PORT || '',
  account: {
    addr: process.env.ADDRESS,
    sk: new Uint8Array(process.env.SK.split(','))
  }
} _//cc.

### CODE REVIEW

**Review by CoreValExtreme — COMMENTED** [review]
I've copy and pasted changes.

File: .env [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+algodTOKEN: {
+  'X-API-Key':process.env.API_KEY || "
+  },
+  algodServer: process.env.ALGO_SERVER||",
+  indexerServer: process.env.INDEX_SERVER||",
+  algodPort: process.env.ALGO_PORT||",
+  account: {
+   addr:process.env.ADDRESS,
+   sk: new  Uint8Array(process.env.SK.split(','))
+     }
```
Comment (CoreValExtreme) [inline .env:19]: [ Git.hub.6%apy [ .env//

**Comment by jasonpaulos** [comment]
@CoreValExtreme can you please explain the purpose of this change and why it belongs in the main repo?

**Comment by CoreValExtreme** [comment]
My apologies if I incorporated wrongly. I can't explain. I've been trying
to correct issues as they come in. What d o I need to do to correct the
mistake?

**Comment by CoreValExtreme** [comment]
I did follow the instructions of making others aware of versions used.

**Comment by jasonpaulos** [comment]
Closing because this doesn't appear to be relevant. Feel free to continue the work on your fork, but I don't see why this should be merged into the main repo.

**Pull request closed without merging**